.github/workflows/ci.yml

```diff
@@ -494,7 +498,7 @@ jobs:
           contains(matrix.modules, 'trino-delta-lake') && contains(matrix.profile, 'cloud-tests') &&
           (env.ABFS_ACCOUNT != '' || env.ABFS_CONTAINER != '' || env.ABFS_ACCESSKEY != '' || env.AWS_ACCESS_KEY_ID != '' || env.AWS_SECRET_ACCESS_KEY != '')
         run: |
-          $MAVEN test ${MAVEN_TEST} -P cloud-tests -pl :trino-delta-lake \
+          $MAVEN test ${MAVEN_TEST} format('-P {0}', matrix.profile) -pl :trino-delta-lake \
```

Comment:
`${{ }}` is missing. https://github.com/trinodb/trino/pull/14087/commits/b319b127bfa00a405e6053591bddcb20e9d23462